XamarinCommunityToolkit/XamarinCommunityToolkit.csproj

```diff
@@ -0,0 +1,107 @@
+        <AssemblyName>XamarinCommunityToolkit</AssemblyName>
+        <RootNamespace>XamarinCommunityToolkit</RootNamespace>
+        <PackageId>XamarinCommunityToolkit</PackageId>
+        <PackageIconUrl>
+        </PackageIconUrl>
+        <Summary>Community-created toolkit with common Xamarin converters, effects, behaviors etc.</Summary>
+        <PackageTag>xamarin,ios,android,uwp,xamarin.forms,effects,controls,converters,animations,toolkit,kit,communitytoolkit,xamarincommunitytoolkit,watchos,tvos,tizen</PackageTag>
+        <Title>XamarinCommunityToolkit</Title>
+        <Description>The Xamarin Community Toolkit is a collection of Animations, Behaviors, Converters, and Effects for mobile development with Xamarin.Forms. It simplifies and demonstrates common developer tasks building iOS, Android, and UWP apps with Xamarin.Forms.</Description>
+        <Product>$(AssemblyName) ($(TargetFramework))</Product>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+        <AssemblyFileVersion>1.0.0.0</AssemblyFileVersion>
+        <Version>1.0.0</Version>
+        <PackageVersion>$(Version)$(VersionSuffix)</PackageVersion>
+        <Authors>Microsoft</Authors>
+        <Owners>microsoft</Owners>
+        <NeutralLanguage>en</NeutralLanguage>
+        <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+        <RepositoryUrl>https://github.com/xamarin/XamarinCommunityToolkit</RepositoryUrl>
+        <PackageReleaseNotes>
+            <!-- See if we can host these on MS docs? -->
+        </PackageReleaseNotes>
+        <DefineConstants>$(DefineConstants);</DefineConstants>
+        <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
+        <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <PackageProjectUrl>https://github.com/xamarin/XamarinCommunityToolkit</PackageProjectUrl>
+        <DebugType>portable</DebugType>
+        <Configurations>Debug;Release</Configurations>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)'=='Debug' ">
+        <DebugSymbols>true</DebugSymbols>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)'=='Release' And '$(OS)' == 'Windows_NT' ">
+        <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <!-- Embed source files that are not tracked by the source control manager in the PDB -->
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+        <!-- Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
+        <IncludeSymbols>true</IncludeSymbols>
+        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
I think snupkg's aren't supported for Xamarin at this point. I don't think it will do any harm though. Just leaving this here as a reminder when things don't work we might need to look here.